### ISSUE TITLE
Reintroducing old security flaws

### ISSUE THREAD

**Issue by wireghoul** [MongoDB-Rox/phpMoAdmin-MongoDB-Admin-Tool-for-PHP#30]
Hello,

I noticed that you reverted the security patch from in version 1.1.5, thus commit #d9f34eebffb8efb4dfe2ff8e429b97f7e16f0999 reintroduced remote code execution again.

This issue has been assigned the following identifier: CVE-2018-3816 and I hope you can resolve this issue properly this time.

**Comment by S4lt5** [comment]
Yeah, please don't use this software. It looks like the dev quietly reintroduced vulns with no documentation.
